src/assets/custom/css/_sass/_training-hero.scss

```diff
@@ -0,0 +1,149 @@
+.training-hero__background-image {
+  background-size: cover;
+  opacity: 0.9;
+  position: relative;
+  width: 100%;
+
+  @include small {
+    height: 216.27px;
+  }
+
+  @include medium-and-large {
+    height: 433px;
+  }
+
+  @include extra-large {
+    height: 519.5px;
+  }
+}
+
+$background-block-height: ($vertical-overlap *  4);
+.training-hero__lower-background-block {
+  background-color: $slate;
+  position: relative;
+
+  @include small {
+    height: 2px + $background-block-height;
+  }
+
+  @include medium {
+    height: 6px + $background-block-height;
```

Comment:
I'm not clear why we need to amend the height in this manner? I think we could simplify this when we re-arrange the HTML markup.